data_prep/generate_climos.py

```diff
@@ -5,37 +5,22 @@
 
 from argparse import ArgumentParser
 from datetime import datetime
-from tempfile import NamedTemporaryFile
 
 from cdo import Cdo
-from netCDF4 import Dataset, num2date, date2num
+from netCDF4 import date2num
 from dateutil.relativedelta import relativedelta
 
-from Cmip5File import Cmip5File, ClimdexFile
+from util import s2d, ss2d, d2ss, d2s
```

Comment:
We're in some kind of github time warp here. If you look on the Files Changed tab, you will see that is exactly what I did.